docs/source/release_notes/release-notes.rst

```diff
@@ -162,7 +141,26 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2093>`__)
 
     *    - |fixed|
-         - Added backwards compatibility for passing row values into AtlasConsole functions
+         - AtlasDB clients now retry lock requests if the server lose leadership while the request is blocking.
+           In the past, this scenario would cause 500 errors that were not retried by the client.
```

Comment:
Should probably also say that the server now returns 503s (relevant for people that use lock service on their own?)

Also nit - if the server _loses_ leadership